tests/petab_test_suite/conftest.py

```diff
@@ -0,0 +1,50 @@
+"""Pytest configuration for PEtab test suite"""
+
+from typing import List
+import re
+import sys
+import petabtests
+
+
+def parse_selection(selection_str: str) -> List[int]:
```

Comment:
is this used?

Reply:
Yes, allows you to run something like `pytest --petab-cases=7,8,9-12`

Reply:
so this will allow to only run the tests that amici should support. ok!